lib/devise_token_auth/controllers/helpers.rb

```diff
@@ -90,14 +95,15 @@ def log_process_action(payload)
       #     Admin
       #
       #   Generated methods:
-      #     authenticate_user!  # Signs user in or 401
-      #     authenticate_admin! # Signs admin in or 401
-      #     user_signed_in?     # Checks whether there is a user signed in or not
-      #     admin_signed_in?    # Checks whether there is an admin signed in or not
-      #     current_user        # Current signed in user
-      #     current_admin       # Current signed in admin
-      #     user_session        # Session data available only to the user scope
-      #     admin_session       # Session data available only to the admin scope
+      #     authenticate_user!                   # Signs user in or 401
+      #     authenticate_admin!                  # Signs admin in or 401
+      #     user_signed_in?                      # Checks whether there is a user signed in or not
+      #     admin_signed_in?                     # Checks whether there is an admin signed in or not
+      #     current_user                         # Current signed in user
+      #     current_admin                        # Current signed in admin
+      #     user_session                         # Session data available only to the user scope
+      #     admin_session                        # Session data available only to the admin scope
+      #     render_authenticate_#{mapping}_error # Render error if
```

Comment:
@olleolleolle Thank you for your review.
You are right. I will modify the comment soom.